lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -243,6 +245,19 @@ spec = do
             , "                           '<amount>@<address>'"
             ]
 
+        ["transaction", "list", "--help"] `shouldShowUsage`
+            [ "Usage:  transaction list [--port INT] WALLET_ID [--after TIME]"
+            , "                         [--before TIME]"
+            , "  List the transactions associated with a wallet."
+            , ""
+            , "Available options:"
+            , "  -h,--help                Show this help text"
+            , "  --port INT               port used for serving the wallet"
+            , "                           API. (default: 8090)"
+            , "  --after TIME             specifies an earliest time"
+            , "  --before TIME            specifies a latest time"
```

Comment:
It would be nice to add info about the time format (ISO-8601 UTC) and maybe exemplary value.

Reply:
That's a good idea. I'll add it.

Reply:
@piotr-iohk Added:

```
transaction list --help
Available options:
  -h,--help                Show this help text
  --port INT               port used for serving the wallet API. (default: 8090)
  --after TIME             specifies an earliest time (ISO 8601 format: basic or extended)
  --before TIME            specifies a latest time (ISO 8601 format: basic or extended)
```